studies/migrations/0076_update_external_study_type.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.0.14 on 2021-09-24 14:19
+
+from django.db import migrations
+
+
+def update_config(apps, schema_editor):
+    StudyType = apps.get_model("studies.StudyType")
+    external = StudyType.objects.get(name="External")
+    external.configuration["metadata"]["fields"] = [
+        {"name": "url", "value": "", "input_type": "url"}
+    ]
+    external.save()
+
+
+def revert_config(apps, schema_editor):
+    StudyType = apps.get_model("studies.StudyType")
+    external = StudyType.objects.get(name="External")
+    external.configuration["metadata"]["fields"] = [
+        {
+            "name": "url",
+            "value": "",
+            "input_type": "url",
+        },
+        {
+            "name": "scheduled",
+            "value": False,
+            "input_type": "checkbox",
+        },
+    ]
+    external.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("studies", "0075_remove_exit_url"),
+    ]
+
+    operations = [migrations.RunPython(update_config, reverse_code=revert_config)]
```

Comment:
That's true!  I didn't merge them as these changes were asked for after being deployed to staging.  I am reluctant to merge migrations after they're in the wild.

Reply:
Oh, good point about them already having been applied to the staging db. If there were a *lot* here I think it'd be worth learning more about messing around directly with the staging db to rewind, but with just a few makes sense to me to get it out the door rather than try to pretty up the migration history :)